specs/block-directory.test.js

```diff
@@ -62,6 +67,10 @@ describe( `Block Directory Tests`, () => {
 
 			const resp = await finalResponse.json();
 
+			// Determine the loaded assets, store it for the next test.
+			freshScripts = await getAllLoadedScripts();
```

Comment:
We could also use the scripts that were returned from the api in the `assets` array in this check as opposed to crawling the DOM. It may help us compare the 'final' block assets and what the API thinks the block's assets are, although I'm struggling to find the difference.

Reply:
The reason here is that it diff's the loaded assets..
1. This should be all the block editor assets, all the wp-admin js, etc. WITHOUT the block assets
2. Installs the block
3. Gets all the assets again, this time with the block assets
4. Diff's to find out what the 'final' asset files are which *should* match what the API thinks, but in the case of say, waves, won't